tests/resource/sample_functions.py

```diff
@@ -0,0 +1,28 @@
+def sample_compare_two_int_lists(list1, list2):
+    if len(list1) != len(list2):
+        return False
+
+    for i, element1 in enumerate(list1):
+        element2 = list2[i]
+        are_elements_equal = sample_compare_integers(element1, element2)
+        if not are_elements_equal:
+            return False
+    return True
+
+
+def sample_compare_integers(value1, value2) -> bool:
```

Comment:
did you mean to add the type hint here? or this even matter?

Reply:
It does not really matter, but I added them just for completeness.

Reply:
Ok :)